tests/test_ast.py

```diff
@@ -377,7 +387,8 @@ def test_isr_def_empty():
 def test_isr_def():
```

Comment:
this is the test that was failing (after converting it to have multiple statements).
a similar test for functions would also fail.

the test i added above (`test_let_multistatement`) never failed.